dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/task/sql/SqlTask.java

```diff
@@ -355,9 +355,8 @@ private PreparedStatement prepareStatementAndBind(Connection connection, SqlBind
         }
         Map<Integer, Property> params = sqlBinds.getParamsMap();
         if(params != null){
-            for(Integer key : params.keySet()){
-                Property prop = params.get(key);
-                ParameterUtils.setInParameter(key,stmt,prop.getType(),prop.getValue());
+            for(Map.Entry<Integer, Property> entry: sqlBinds.getParamsMap().entrySet()) {
```

Comment:
"sqlBinds.getParamsMap()" can be replaced with  "params.entrySet()" 

since there is alreay a  params.

Reply:
ok